src/microsoft-trydotnet-editor/src/index.css

```diff
@@ -2,6 +2,7 @@ body {
 	width: auto;
 	height: auto;
 	margin: 0;
+	overflow: hidden;
```

Comment:
If the content is sized correctly, I would think we shouldn't need to do this.

Reply:
It seems that the issue is not in the content.
Here is the animated gif showing the behavior WITHOUT 'overflow: hidden'
![ScrollBar](https://github.com/dotnet/try/assets/510598/4898f9e8-647c-4caf-8b94-5ae4aaed3603)

Reply:
Let's find some time to discuss.